packages/clients/address-service-client/index.js

```diff
@@ -31,7 +31,7 @@ function createTestInstance (existingItem = null) {
  * @returns {MockedAddressServiceClient|AddressServiceClient}
  */
 function createInstance (testItem) {
-    return conf.NODE_ENV === 'test' || get(conf, 'ADDRESS_SERVICE_CLIENT_MODE') === 'fake'
+    return conf.NODE_ENV === 'test' || get(conf, 'ADDRESS_SERVICE_CLIENT_MODE', 'fake') === 'fake'
```

Comment:
Are you sure these changes are needed?

Reply:
Added backward-compatible change in another PR (outdated)